### PR TITLE
data: hzpeng57

### PR DESCRIPTION
**Pull request by hzpeng57**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7835"></a>

### CODE REVIEW

File: data/hzpeng57.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "tags": ["Typescript", "Vue.js"],
+  "links": [
+    {
+      "name": "GitHub",
+      "url": "https://github.com/hzpeng57",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/hzpeng/"
```
Comment (sandramsc) [inline data/hzpeng57.json:13]: Hi, you have a comma (**,**) missing on **line 13,** rendering the json file invalid.

**Review by sandramsc — CHANGES_REQUESTED** [review]
Hi, you have a comma missing on line 13, rendering the json file invalid.

**Review by sandramsc — CHANGES_REQUESTED** [review]
Hi, you have a comma (**,**) missing on **line 13**, rendering the json file invalid.

**Review by pushkaraj2007 — CHANGES_REQUESTED** [review]
Hello,

On line 13, there appears to be a missing comma, which is causing errors during JSON rendering. In order to ensure the smooth functioning of the project, it is necessary to add a comma at the appropriate position.

Could you add the comma at the end of line 13?

File: data/hzpeng57.json (hunk 1 above)
Comment (SaraJaoude) [inline data/hzpeng57.json:13]: add a comma
```suggestion
      "url": "https://www.linkedin.com/in/hzpeng/",
```

**Comment by SaraJaoude** [comment]
> Hello,
> 
> On line 13, there appears to be a missing comma, which is causing errors during JSON rendering. In order to ensure the smooth functioning of the project, it is necessary to add a comma at the appropriate position.
> 
> Could you add the comma at the end of line 13?

@pushkaraj2007 please avoid duplicating a comment - this was just mentioned above  - and mentioning it again can cause confusion to the author as well as unnecessary extra notifications

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by hzpeng57** [comment]
@SaraJaoude Thank you for correcting my mistake.

**Comment by pushkaraj2007** [comment]
> > Hello,
> > On line 13, there appears to be a missing comma, which is causing errors during JSON rendering. In order to ensure the smooth functioning of the project, it is necessary to add a comma at the appropriate position.
> > Could you add the comma at the end of line 13?
> 
> @pushkaraj2007 please avoid duplicating a comment - this was just mentioned above - and mentioning it again can cause confusion to the author as well as unnecessary extra notifications

@SaraJaoude I'm sorry for duplicating the comment, actually I didn't notice the comment of @sandramsc. I'll definitely take care of this in the future.

Sorry once again!

**Pull request merged**